### PR TITLE
Align help output with manpage 3447

### PR DESCRIPTION
**Pull request by thutterer**
this is a follow-up after #283 

- there were a few formatting things left to do in the manpage.
- the `--help` output was not yet aligned with the manpage changes.
- the line length of `--help` was >120. now we have a spec to make sure its never larger than a 80 chars.

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
@lagartoflojo will review your pull request :dancers: check https://trello.com/c/VhhCQOAN/3447-2-connect-man-pages-implement-tanja-s-feedback
  @lagartoflojo: Please review this pull request using our guidelines:
  * test for acceptance criteria / functionality
  * check if the new code is covered with tests
  * check for unintended consequences
  * encourage usage of the boyscout rule
  * make sure the code is architected in the best way
  * check that no unnecessary technical debt got introduced
  * make sure that no unnecessary FIXMEs or TODOs got added

File: spec/connect/cli_spec.rb [hunk 1]
```diff
@@ -293,9 +293,11 @@
       subject.new(argv)
     end
 
-    it 'output help on help flag' do
+    it 'outputs help on help flag with no line longer than 80 characters' do
       argv = %w{--help}
-      expect_any_instance_of(subject).to receive(:puts)
+      expect_any_instance_of(subject).to receive(:puts) do |option_parser|
+        expect(option_parser.instance_variable_get(:@opts).to_s.split("\n").map(&:length).max).to be <= 80
```
Comment (lagartoflojo) [inline spec/connect/cli_spec.rb]: I find `option_parser.instance_variable_get(:@opts)` very, very ugly... I don't even know why it works.

What do you think of this solution? http://stackoverflow.com/a/16507680/686769 (check the *RSpec < 3.0 (and others)* section, method `capture_stdout`).

Comment (thutterer) [inline spec/connect/cli_spec.rb]: Well, the instance of OptionParser has a `@opts` variable whose `to_s` method builds the help output. not the most obvious thing, I agree :wink:
But do you really insist to have a custom spec helper just for this one occurrence, where the current implementation already does what it should? 
And where your proposed solution does not? :stuck_out_tongue: I just tried it, and it just doesn't capture the output and I don't wanna spend time debugging some random code from StackOverflow.

Comment (lagartoflojo) [inline spec/connect/cli_spec.rb]: I didn't try it either, it was just an idea. So no, I don't insist. Too bad RSpec's own `output` matcher can't be used in this case.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/2/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged f91e1ab074168ee0c87527e38e5a4205ded06f82 into dc44fa5e0a768c466396d4fed9c9ea1f8821dfe8 )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-Connect-Prophet/3/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 7b8b3bbc97c24f87bae129c6945c0dcf962caa65 into dc44fa5e0a768c466396d4fed9c9ea1f8821dfe8 )

**Pull request merged**